### DISCUSSION TITLE
"Pause Auto-Merge" checkbox in dependency dashboard

### DISCUSSION THREAD

**Discussion by Turbo87** [renovatebot/renovate#29699] [Suggest an Idea]
### Tell us more.

It would occasionally be useful to be able to pause/suspend the auto-merging behavior of renovatebot. For example, when a CD pipeline is set up and you are dealing with a bugfix, you might not want automated dependency updates to interfere with your deployments. Having a checkbox in the dependency dashboard to temporarily pause the auto-merging behavior would probably help in such situations.

**Comment by HonkingGoose** [comment]
Do you want a timer?

- [ ] Pause Renovate automerge for 12 hours
- [ ] Pause Renovate automerge for 24 hours

Or a on/off toggle?

- [ ] Pause Renovate automerges when this checkbox is selected

I lean towards giving the users a timed pause button. Users may forget to enable automerging when they're done.

Also should Renovate still try to automerge security critical updates? Or do you really want _no automerges at all_?

**Reply by Turbo87** [reply]
I was thinking on/off toggle, but a timer would probably also work.

> Users may forget to enable automerging when they're done.

my naive first thought was opening an issue while automerging is paused and when the user closes the issue then automerging continues. might be more visible than the small checkbox?



> Also should Renovate still try to automerge security critical updates? Or do you really want _no automerges at all_?

I think I'd expect it to disable automerging in general. if there are security updates then they can still be applied manually by merging the corresponding pull request for example.

**Reply by rarkins** [reply]
I'd prefer it to be an on/off checkbox, non time limited and applies to all PRs

**Reply by secustor** [reply]
I have created an issue https://github.com/renovatebot/renovate/issues/29712
